homework5/ParsingTree/ParsingTree/NumberNode.cs

```diff
@@ -0,0 +1,34 @@
+// <copyright file="NumberNode.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace ParsingTree;
+
+/// <summary>
+/// The node with integer value.
+/// </summary>
+public class NumberNode : Node
+{
+    private readonly int value;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="NumberNode"/> class.
+    /// </summary>
+    /// <param name="value">Node`s value.</param>
+    public NumberNode(int value)
+    {
+        this.value = value;
+    }
```

Comment:
```suggestion
    public NumberNode(int value) => this.value = value;
```
С конструкторами, содержащими инициализацию одного поля, на самом деле, также можно